src/components/Checkout/SocialShareV2.vue

```diff
@@ -47,11 +47,11 @@
 				class="social__btn social__btn--link tw-text-link tw-border-tertiary tw-border"
 				:class="copyStatus.class"
 				:disabled="copyStatus.disabled"
-				v-kv-track-event="['thanks', 'Social-Share-Lightbox', 'click-Copy-link-share']"
+				v-kv-track-event="['thanks', 'Social-Share-Lightbox', 'click-Copy-link-share', selectedLoan.id]"
```

Comment:
I think we need to break out a computed property for use in the template that returns `selectedLoanId` in a safe way in case it's not present. This would fix the potential undefined error down in the `handleFacebookResponse` as well.

Reply:
I don't believe this will throw an error. the computed for `selectedLoan` will return an empty object if theres no loan for some reason. So accessing `.id` on this empty object will just send undefined to our track event function in analytics, which the function can handle